packages/docusaurus-theme-classic/src/theme/DocSidebar/Desktop/styles.module.css

```diff
@@ -24,8 +20,6 @@
 
   .sidebarHidden {
     opacity: 0;
-    height: 0;
-    overflow: hidden;
     visibility: hidden;
   }
```

Comment:
`.sidebar` will not shrink in size on collapse anymore. It will just become invisible. This preserves the height.

Reply:
thanks, looks better indeed 👍 

Wonder if it affects accessibility and keyboard navigation? Is it possible now to navigate with the keyboard (tabs) on these hidden elements?

Reply:
@slorber That's a good point. I haven't tested keyboard navigation on hidden elements.